src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -190,9 +194,18 @@ private void handleExit() {
         primaryStage.hide();
     }
 
-//    public ContactListPanel getContactListPanel() {
-//        return contactListPanel;
-//    }
+    private void changeTabOnCommandEntered(String commandText) {
+        int typeLetter = commandText.charAt(0);
+        if (typeLetter == 'T' || typeLetter == 'L') {
+            tabPane.getSelectionModel().select(0);
+        }
+        if (typeLetter == 'M') {
+            tabPane.getSelectionModel().select(1);
+        }
+        if (typeLetter == 'C') {
+            tabPane.getSelectionModel().select(2);
+        }
+    }
```

Comment:
Perhaps can change this to a switch clause? Also instead of the firstChar maybe use Class.TYPE?
so like switch (TYPE):
   case Task.TYPE:
   case  Lesson.TYPE:
         ...........
    case Module.TYPE:
          ...........
    case Contact.TYPE:
           ....